watcher/fsevents.c

```diff
@@ -279,6 +280,9 @@ static void *fsevents_thread(void *arg)
   // Block until fsevents_root_start is waiting for our initialization
   pthread_mutex_lock(&state->fse_mtx);
 
+  state->attempt_resync_on_user_drop =
+      cfg_get_bool(root, "fsevents_try_resync", false);
```

Comment:
Hmm.

* Is the plan here to switch the default to `true` at some point?
* Will there come a time when we can make this config option go away entirely?